tico/config/v1.py

```diff
@@ -20,6 +20,7 @@
 @dataclass
 class CompileConfigV1(CompileConfigBase):
```

Comment:
Should I create `CompileConfigV2` instead of updating this?

Reply:
It's okay to update this class.